streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -331,19 +331,22 @@ private[streaming]
 object FileInputDStream {
 
   /**
-   * Minimum duration of remembering the information of selected files. Files with mod times
-   * older than this "window" of remembering will be ignored. So if new files are visible
-   * within this window, then the file will get selected in the next batch.
+   * Minimum duration of remembering the information of selected files. Defaults to 1 minute.
+   *
+   * Files with mod times older than this "window" of remembering will be ignored. So if new
+   * files are visible within this window, then the file will get selected in the next batch.
    */
-  private val MIN_REMEMBER_DURATION = Minutes(1)
+  private val minRememberDurationMin = Minutes(new SparkConf()
```

Comment:
@srowen I'm a little confused. Do you mean something like: "inside ``private[streaming] class FileInputDStream[K, V, F <: NewInputFormat[K,V]]`` class create a ``private val conf = new SparkConf()`` and then use that ``conf`` inside ``private[streaming] object FileInputDStream``"?

Or something totally different?